values.yaml.tpl

```diff
@@ -19,7 +19,7 @@ deployments:
     version: ${JURISDICTION_API_VERSION}
   mds-jurisdiction-service:
     version: ${JURISDICTION_SERVICE_VERSION}
-  mds-policy-author:
+  mds-policy-author-api:
```

Comment:
👇🏻 `POLICY_AUTHOR_API_VERSION`?

Reply:
Oops, already made that change, forgot to push :)